### DISCUSSION TITLE
Rust extra: Cargo.toml is messed on save

### DISCUSSION THREAD

**Discussion by osenvosem** [LazyVim/LazyVim#4180] [Q&A]
Every time I edit Cargo.toml it is messed on save moving lines without any logic. Here I added chrono dependency and this is what I got:

<img width="834" alt="Screenshot 2024-07-25 at 13 15 45" src="https://github.com/user-attachments/assets/39766bed-e710-4f6d-914f-0ff26d382949">

No config for the Rust extra components changed. How to solve?

**Comment by ogios** [comment]
i used to have this issue, and i have to wait until every dependency's version check is done, then i can modify it without messing up the cargo toml.

but i do not have have this issue now.
